docs/layers/base-layer.md

```diff
@@ -1,27 +1,28 @@
 # Layer Base Class
 
 The `Layer` class is the base class of all deck.gl layers, and it provides
-a number of **base properties** which are availabe in all layers.
+a number of **base properties** availabe in all layers.
 
 ## Static Members
 
 #### `layerName` (static, string, required)
 
-Tthis static property should contain the name of the layer,
+This static property should contain the name of the layer,
 typically the name of layer's class (it cannot reliably be autodeduced in
 minified code). It is used as the default Layer id as well as for debugging
 and profiling.
 
 #### `defaultProps` (static, object, optional)
 
-Layers are strongly encouraged to define a `defaultProps` static member
-listing their props and default values, rather than programmatically
-declaring the props in constructor parameters etc. Using `defaultProps` improves
+All deck.gl layers define a `defaultProps` static member listing their props and
+default values, rather than programmatically declaring the props in their
```

Comment:
Should we just drop the "rather than programmatically ..." as this is outdated)?